packages/zoe/test/unitTests/contracts/test-atomicSwap.js

```diff
@@ -16,18 +16,18 @@ const filename = new URL(import.meta.url).pathname;
 const dirname = path.dirname(filename);
 
 const atomicSwapRoot = `${dirname}/../../../src/contracts/atomicSwap.js`;
+const atomicSwapWithFeeRoot = `${dirname}/../../../src/contracts/atomicSwapWithFee.js`;
```

Comment:
It would be better to create a separate test so this one can continue to test the original atomic swap.